.travis.yml

```diff
@@ -5,7 +5,7 @@ before_install:
   - |
     if [[ "$(ruby -e 'puts RUBY_VERSION')" != 1.* ]]; then gem update --system; fi
 rvm:
-  - 1.9.3
-  - 2.3.3
-  - 2.4.0
+  - 2.3.8
+  - 2.4.5
+  - 2.5.3
```

Comment:
Just a note - Ruby 2.0 is EOL in upstream, but it's still part as Red Hat Enterprise Linux 7 and all its clones with guaranteed support of 10+ years. If there is any chance of keeping 2.0.0 it's not a bad idea to keep it for another few years (2024 is EOL for Ruby 2.0 in CentOS7, a bit later for RHEL7 extended support).

What I am trying to say, there is a lot of software running just fine on Ruby 2.0.x and if there is not strong reason to ditch it, I'd vote keeping it.

Reply:
Thank you for your feedback. Some thoughts on this:
- 2.0 doesn't support required keyword arguments, so we'd need at least 2.1. We could side-step this by not using them, but I much prefer how the interface feels with them.
- Our test suite passes in Ruby 2.1.10. So, while we're not running CI on it, you should be able to use this in 2.1